card/mod/05_standard/set/all/rich_html/editing.rb

```diff
@@ -38,8 +31,9 @@ def default_new_args args
       end
     end
     args[:optional_name_formgroup] ||= :hide
+  end
 
-    # type field
+  def default_new_args_for_type_field args
```

Comment:
Assignment Branch Condition size for default_new_args_for_type_field is too high. [21.86/15]